### DISCUSSION TITLE
How can we add GSSoC'24 earned badge to Github profile?

### DISCUSSION THREAD

**Discussion by vaishali-sharma-20** [sanjay-kv/Open-source-Practice#990] [Q&A] [answered]
**Is it possible to show GSSoC'24 badges to the GitHub Profile in the readme file?**
If so, please let me know how I can add it to my profile.

**Comment by sanjay-kv** [comment, marked as answer]
you cant trasfer those badge to github. 
![image](https://github.com/GSSoC24/Open-source-Practice/assets/30715153/528ec446-6b35-45ec-9b91-3c0abac8ab0d)
github can only allow this badges.

https://github.com/GSSoC24/Contributor/discussions/11
